1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,6 +1,27 @@
-function setAlarm() {}
+ 
+ 
 
-// DO NOT EDIT BELOW HERE
+ function setAlarm() {
+  let getInputValue = document.querySelector("#alarmSet").value; 
+  console.log(getInputValue);
+  let title = document.querySelector("#timeRemaining");
+  console.log(title);
+  title.innerHTML=`Time Remaining: 00:${getInputValue}`;
+
+  let timeLeft = getInputValue  
+  const Interval = setInterval(()=>{
+    timeLeft--;
+       document.querySelector("#timeRemaining").innerHTML=`Time Remaining: ${timeLeft}`;
+      if(timeLeft === 0){
+        playAlarm();
+        clearInterval(Interval)
+      }
+    },1000);
+  }
+    
```

Comment:
Hi Zobeir, your code is very neat. well done.  I wonder, will it work more efficiently if you move line 16 to line 17 and line 17 to line 16?? I am just thinking of JS reading from top to bottom and whether it will run better if it plays alarm first, then clear once it is stopped? I am just wondering this....that is all. Well done Zobeir.